_data/alekao/alekao.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Raymond Liu
+institution: Lawrence Park Institute Collegiate
+profile_pic: RL.jpg
```

Comment:
image pathname seems incorrect to me, please change it from `RL.jpg` to `alekao.jpg`